docs/plugin-development.rst

```diff
@@ -618,66 +621,35 @@ Each type needs to be installed differently due to how node manages external pac
+  The final alternative for Webpack-built plugins is to set the ``npm.install``
+  configuration property to ``true``; this will cause the build system to run
+  ``npm install`` in the plugin directory.
 
-      import foobar from 'girder_plugins/MY_PLUGIN/node/foobar'
+  When you use the `import` directive within your plugin code, for example:
 
-  would import the default value from NPM dependency ``foobar`` as installed
-  in ``MY_PLUGIN``'s dedicated ``node_modules_MY_PLUGIN`` directory. This is mainly
-  useful if you need a different version of a package already in use by Girder
-  core, or if for any other reason you prefer to keep your plugin dependencies
-  isolated. By default, the ``localNodeModules`` is set to ``false`` and the
-  dependencies will be installed to Girder's own ``node_modules`` directory.
+  .. code-block:: javascript
 
-  The final alternative for Webpack-built plugins is to set the ``npm.install``
-  configuration property to ``true``; this will cause the build system to run
-  ``npm install`` in the plugin directory. This may have certain benefits for
-  plugin development, such as allowing plugin sources to import modules without
-  the alias prefix as described above (though, this alias would still be
-  available for use by other plugins that want to access your plugin's
-  dependencies). Additionally, if your plugin is installed without using
-  symlinks, then you will still have access to Girder's Node dependencies (see
-  this [GitHub conversation](https://github.com/nodejs/node/issues/3402) for a
-  discussion of why symlinked directories will not allow for the usual Node
-  import semantics).
-
-  If instead you are using a custom Grunt build with a Gruntfile, the
-  dependencies should be installed into your plugin's **node_modules** directory
-  by providing a `package.json <https://docs.npmjs.com/files/package.json>`_
-  file just as they are used for standalone node applications.  When such a file
-  exists in your plugin directory, ``npm install`` will be executed in a new
-  process from within your package's directory.
-
-- Build time dependencies that your Grunt tasks rely on to assemble the sources
-  for deployment need to be installed into Girder's own **node_modules** directory.
-  These dependencies will typically be Grunt extensions defining extra tasks used
-  by your build.  Such dependencies should be listed under ``grunt.dependencies``
-  as an object (much like dependencies in **package.json**) inside your
-  **plugin.json** or **plugin.yml** file.
+      import foobar from 'foobar';
 
-  .. code-block:: json
+   The build process will search for the ``'foobar'`` module in the following locations, in order:
 
-      {
-          "name": "MY_PLUGIN",
-          "grunt": {
-              "dependencies": {
-                  "grunt-contrib-concat": ">=1.0.1"
-              }
-          }
-      }
+   1. The plugin's local modules directory that is created if ``npm.localNodeModules`` has
```

Comment:
This documentation appears to be under the javascript code block, and renders incorrectly if you go to View in GitHub.